Formula/hyperkit.rb

```diff
@@ -0,0 +1,120 @@
+class Hyperkit < Formula
+  desc "Lightweight virtualization hypervisor for MacOS"
+  homepage "https://github.com/moby/hyperkit"
+  url "https://github.com/moby/hyperkit.git",
+    :tag => "v0.20170425",
+    :revision => "a9c368bed6003bee11d2cf646ed1dcf3d350ec8c"
+
+  bottle do
```

Comment:
>Shouldn't Test Bot just overwrite this section?

The first commit adding a new formula never contains a preexisting bottle block.

 >Also, if I wanted to provide bottles it would be permitted wouldn't it?

Only bottles built by BrewTestBot are permitted in Homebrew Core, so no it would not be permitted.